### PR TITLE
feat(pytest): add support for calling patch_all via the plugin

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Review by P403n1x87 — COMMENTED** [review]
Can this new feature be tested?

**Comment by jd** [comment]
Let me look into it.

File: tests/contrib/pytest/test_pytest.py [hunk 1]
```diff
@@ -29,6 +29,41 @@ def subprocess_run(self, *args):
...
+        )
+        file_name = os.path.basename(py_file.strpath)
+        rec = self.inline_run("--ddtrace-patch-all", file_name)
+        rec.assertoutcome(passed=1)
+        spans = self.pop_spans()
+
+        assert len(spans) == 0
+
+    def test_patch_all_init(self):
+        """Test withddtrace-patch-all via ini."""
```
Comment (P403n1x87) [inline tests/contrib/pytest/test_pytest.py:50]:
```suggestion
        """Test with ddtrace-patch-all via ini."""
```

File: tests/contrib/pytest/test_pytest.py [hunk 2]
```diff
@@ -29,6 +29,41 @@ def subprocess_run(self, *args):
...
+        self.testdir.makefile(".ini", pytest="[pytest]\nddtrace-patch-all=1\n")
+        py_file = self.testdir.makepyfile(
+            """
+            import ddtrace.monkey
+
+            def test_patched_all():
+                assert ddtrace.monkey._PATCHED_MODULES
+        """
```
Comment (P403n1x87) [inline tests/contrib/pytest/test_pytest.py:58]:
```suggestion
            """
```

Comment (jd) [reply tests/contrib/pytest/test_pytest.py:58]: If you want, but it's not the convention in this file.
I don't really care anyhow.

Comment (P403n1x87) [reply tests/contrib/pytest/test_pytest.py:58]: it's just a suggestions since the indentation looked inconsistent 🙂

**Review by Kyle-Verhoog — DISMISSED** [review]

**Review by P403n1x87 — DISMISSED** [review]

**Review by Yun-Kim — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**